README.md

```diff
@@ -206,6 +206,26 @@ to switch to the vty console frontend optimized for screen readers, run
     cabal run -fvty LambdaHack
 
 
+Building and running under Windows with stack
+---------------------------------------------
+I managed to build and run it by using the stack build rather than the cabal build. I installed sdl2, sdl-ttf, and sdl-mixer using the technique here (https://www.reddit.com/r/haskellgamedev/comments/4jpthu/windows_sdl2_is_now_almost_painless_via_stack/) _for each library_, not just for sdl2.
+
+Then
+
+    stack update
+    stack build
+
+You won't be able to run the game yet because the sdl dlls are missing. These can be grabbed from the binary distributions at the very bottom of the release changelists. https://github.com/JamieFristrom/LambdaHack/tags
```

Comment:
I think the libraries not being visible can be fixed by extending PATH, as here:

https://github.com/LambdaHack/LambdaHack/blob/master/appveyor.yml#L34

(only a small part of this is really needed; can't tell which)